### PR TITLE
(Improve order flow) Refactor validation services

### PR DESCRIPTION
**Pull request by islean**
## Description

Closes https://github.com/Clinical-Genomics/improve-order-flow/issues/108

### Added

-

### Changed

- Use the same OrderValidationService for all order types

### Fixed

-

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b [THIS-BRANCH-NAME] -a
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

File: cg/server/endpoints/orders.py [hunk 1]
```diff
@@ -272,31 +264,11 @@ def validate_order(order_type: OrderType):
...
-        response = taxprofiler_validation_service.validate(raw_order)
-    elif order_type == OrderType.TOMTE:
-        response = tomte_validation_service.validate(raw_order)
+    rule_set: RuleSet = ORDER_TYPE_RULE_SET_MAP[order_type]
+    model: type[Order] = ORDER_TYPE_MODEL_MAP[order_type]
+    response = order_validation_service.get_validation_response(
+        raw_order=raw_order,
+        rule_set=rule_set,
+        model=model,
+    )
```
Comment (ChrOertlin) [inline cg/server/endpoints/orders.py:273]: ❤️

**Review by ChrOertlin — APPROVED** [review]
Much better! We could rename the workflows folder to validation_rules folder and have all .py file in there named for each workflow. Very minor comment, disregard if you don't agree.

File: cg/server/endpoints/orders.py (hunk 1 above)
Comment (diitaz93) [reply cg/server/endpoints/orders.py:273]: 💓

**Review by diitaz93 — APPROVED** [review]
Awsome 🚀

File: cg/server/endpoints/orders.py (hunk 1 above)
Comment (ChrOertlin) [reply cg/server/endpoints/orders.py:273]: as discussed on the huddle- the rule set and model can be parsed within the service. The service signature can receive just the order_type.

this way the endpoint does not require to know the information of a rule set or model map

Comment (islean) [reply cg/server/endpoints/orders.py]: Changed!

**Pull request merged**